README.md

```diff
@@ -1,71 +1,68 @@
 
-TODO: Insert pic
+k0smotron is a service designed to manage the lifecycle of k0s control planes in a Kubernetes (any distro) cluster. By running the control plane on a k8s cluster we can enjoy and leverage the high availability and auto-healing functionalities of the management cluster.
 
 ## Use cases
 
-### CI/CD
+### Development and CI/CD
 
-Often when running integration and end-to-end testing for your software running in Kubernetes you need somewhat temporary clusters in CI. Why not leverage the true flexibility and create those clusters on-demand using k0smotron. Creating a controlplane is as easy as creating a custom resource, so is the deletion of it. No more long living snowflake clusters for CI purposes.
+In the process of continuous integration and end-to-end testing, a temporary Kubernetes Cluster is needed. With k0smotron, these clusters can be created on demand in a declarative way and thus integrated into the existing CI process with ease. This avoids cluster sprawl and long-living snowflake clusters.
 
-### Edge
+### Edge Container Management
 
-Running Kubernetes on the network edge usually means running in low resource infrastructure. What this often means is that setting up the controlplane is either a challenge or a mission impossible. Running the controlplane on a existing cluster, on a separate dedicated infrastructure, removes that challenge and let's you focus on the real edge. 
+Kubernetes at the edge typically comes with the requirement of a low resource footprint. As a result, clusters with distributed roles are more challenging and a lot of single-node clusters are created. Managing the large number of clusters confronts us with almost impossible tasks. 
 
-Running on the edge often also means large number of clusters to manage. Do you really want to dedicate nodes for each cluster controlplane and manage all the infrastructure for those?
+Offloading the control plane means that the persistence layer of the cluster can run on dedicated hardware and workloads can be managed at the edge on devices dedicated to their purposes. With k0smotron worker nodes get ephemeral.
 
-## Installation
+### Multi-cloud Cluster LCM
 
-```
-kubectl apply -f https://docs.k0smotron.io/stable/install.yaml
-```
+A multi-cloud strategy is essential these days. But managed Kubernetes offerings are often different in versioning or even the built-in tooling.
 
-## Creating a cluster
+With k0smotron, you can run the control plane management cluster in a public or private cloud provider of your choice and the worker nodes in different clouds. With this you have an approach to unified cluster management, providing one flavor on all clouds and saving the costs for the highly available Control Plane in the cloud.
```

Comment:
I'd add that users can also get a homogenized central management over various clouds.

Reply:
Added "homogenized" to it.